ipaserver/plugins/trust.py

```diff
@@ -661,56 +639,51 @@ class trust_add(LDAPCreate):
     range_types = {
         u'ipa-ad-trust': unicode(_('Active Directory domain range')),
         u'ipa-ad-trust-posix': unicode(_('Active Directory trust range with '
-                                        'POSIX attributes')),
+                                         'POSIX attributes')),
                   }
 
     takes_options = LDAPCreate.takes_options + (
         _trust_type_option,
         Str('realm_admin?',
             cli_name='admin',
-            label=_("Active Directory domain administrator"),
-        ),
+            label=_("Active Directory domain administrator")),
```

Comment:
Same for this block

Reply:
All these changes were done because pep8/pylint were complaining